tests/IntegrationTest.php

```diff
@@ -214,7 +214,7 @@ public function provideRenderStimulusController()
                 ],
             ],
             'controllerValues' => [],
-            'expected' => 'data-controller="my-controller" data-my-controller-boolean-value="1" data-my-controller-number-value="4" data-my-controller-string-value="str"',
+            'expected' => 'data-controller="my-controller" data-my-controller-boolean-value="true" data-my-controller-number-value="4" data-my-controller-string-value="str"',
```

Comment:
i've implemented `(string) true|false` for the rendered value. But this "breaks" the previous `1` === `true` output even though false was blank. 

I personally prefer `true|false` over `1|0` as it's clear that the value is not intended to be an `int` / `(string)` number. But would this be considered BC even though the PR is intended to fix `false` bool's?

Reply:
Do not think this is a BC break, because one who defined a Stimulus value as `Boolean`, and the value was true-thy, having it previously rendered as `..-value="1"`, which in Stimulus was correctly interpreted as `true`.

And fixing that the `false` value was not being rendered as `..-value="false"` means imo it's currently incorrect, and of course the exact purpose of your PR ;].